docs/articles/new-expensify/billing-and-subscriptions/Billing-page-coming-soon.md

```diff
@@ -1,6 +1,49 @@
 ---
 title: Billing and Subscriptions
-description: Coming soon
+description: Billing Page Overview
 ---
 
-# Coming Soon
+# Billing Overview
+
+At the beginning of each month, the Billing Owner for the workspace will be billed for the previous month’s activity. 
+Your Expensify bill is determined by: 
+The number of active members in your workspace
+- Whether you have a Collect or Control plan
+- Whether you’re on pay-per-use or an annual subscription
+- Whether you’re using the Expensify Visa® Commercial Card
+- Active members
+
+An active member is anyone who chats, creates, submits, approves, reimburses, or exports a report in Expensify in any given month. This includes Copilots and automated settings.
```

Comment:
"Automated settings", or is it more "automated actions" or "automation settings"?